src/sst/core/componentInfo.h

```diff
@@ -142,10 +142,19 @@ class ComponentInfo
      */
     uint64_t share_flags;
 
+    /**
+     * @return True if parent ports are shared with the child
+     */
     bool sharesPorts() { return (share_flags & SHARE_PORTS) != 0; }
 
+    /**
+     * @return True if parent statistics are shared with with the child
+     */
     bool sharesStatistics() { return (share_flags & SHARE_STATS) != 0; }
 
+    /**
+     * @return True if parent insert statistics are shared with the child
+     */
```

Comment:
Should instead be "True if child can insert its statistics into its parent's statistics set"